### ISSUE TITLE
I got tired of the tech scene, I became an engineer because I liked computers, but once I started working everything revolved around money, did you ever face a similar situation? and how did you deal with it?

### ISSUE THREAD

**Issue by ospfranco** [jasoncwarner/ama#34]

**Comment by jasoncwarner** [comment]
Yes, though I think that is completely natural when something goes from hobby or interest to job. 

I love writing, on many topics or genres. When I do it for me or friends or family, it's one thing. When I had a book deal for a series of fitness books, it was work and it felt very very different. 

My advise in this situation is very simple: 

1. Do not listen to all the people who say follow your passion and it won't be work. IMV this is not how it works in the real world. This is some Tony Robbins level bubble gum wrapper philosophy that doesn't work except for the person raking in the cash from followers hoping to have what the 'philosopher' has. 

2. Realize that work is work and money is needed to make the world go round, pay the bills, and generally go about life. Work is work. Your goal with a job is to make money. If you weren't being paid by your company it is highly unlikely you would be doing that job for/with those people in that circumstance. It just is. 

3. Work is work. There are very few people in the world that can (and should) find community, fulfillment, meaning, happiness from work. In our day and age, it feels that work has started to become 'all things to all people' and we need work to fulfill other needs we have. 

I *strongly* suggest you not look at it this way. That road leads to ruin. 

Work is work. 

4. If you like to program, find a thing that gets that feeling back. I like to program too, but the pressure to perform with twitter and github and everything else can be high. Do it for you, not anyone else. Don't want to write tests for your side project? Don't. Want to use some new framework or thing and then throw it away? please do. Don't perform for others. Be you and find you and then do it for you. 

that's it for now.
